packages/gestalt/src/PageHeader.test.js

```diff
@@ -0,0 +1,16 @@
+// @flow strict
+import React from 'react';
+import { create } from 'react-test-renderer';
+import PageHeader from './PageHeader.js';
+
+describe('PageHeader', () => {
+  it('renders', () => {
+    const tree = create(<PageHeader title="Settings" />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders subtext', () => {
+    const tree = create(<PageHeader title="Settings" subtext="5 followers" />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
I guess this is enough. Considering there are so many combinations of primary/secondary actions.